docker/llm/serving/cpu/docker/README.md

```diff
@@ -29,7 +28,74 @@ sudo docker run -itd \
         $DOCKER_IMAGE
 ```
 
-
 After the container is booted, you could get into the container through `docker exec`.
 
 To run model-serving using `BigDL-LLM` as backend, you can refer to this [document](https://github.com/intel-analytics/BigDL/tree/main/python/llm/src/bigdl/llm/serving).
+Also you can set environment variables and start arguments while running a container to get serving started initially. You may need to boot serveral containers to support. 
```

Comment:
you may need to boot `several` containers to support the serving service.

typo + more details